### PR TITLE
Swapnil_Assignment_57_to_68

### PR DESCRIPTION
**Pull request by swapnilhole11690**
@AnvitP @tejasphase22 
image
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

File: src/swapnil/Assignment63.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+			revWord = revWord + Word.charAt(i);
+		}
+
+		return revWord;
+	}
+
+	String[] revArray(String[] input) {
+		String[] output = new String[input.length];
+		for (int i = 0, j = input.length - 1; i < input.length; i++, j--) {
+			String ABC = revWord(input[i]);
```
Comment (tejasphase22) [inline src/swapnil/Assignment63.java:19]: No need of this line.
You can directly write like following: 
`output[j] = revWord(input[i]).toLowerCase();`

File: src/swapnil/Assignment65.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+			if (!Character.isDigit(ch)) {
+				output = output + ch;
+			}
+		}
+		return output;
+	}
+
+	String getFirstLetterUpparCase(String word) {
+		int j = word.length() - 1;
+		return word.substring(0, 1).toUpperCase() + word.substring(1, j) + word.substring(j).toLowerCase();
```
Comment (tejasphase22) [inline src/swapnil/Assignment65.java:28]: `return word.substring(0, 1).toUpperCase() + word.substring(1, j) + word.substring(j).toLowerCase();`
Here, can you explain `word.substring(1, j)` part?
I can understand here that:
`word.substring(0, 1).toUpperCase()` - Makes 1st letter into uppercase
`word.substring(j).toLowerCase()` - Makes last letter into lowercase

File: src/swapnil/Assignment66.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	}
+
+	String[] getArray(String[] input) {
+		String[] output = new String[input.length];
+		for (int i = 0; i < input.length; i++) {
+			if (getDigitWord(input[i])) {
+				output[i] = getWordWithoutDigit(input[i]);
+			} else {
+				output[i] = revWord(input[i]);
+			}
```
Comment (tejasphase22) [inline src/swapnil/Assignment66.java:45]: I'm not getting this part.
Please explain the code line by line in the proper way.

**Review by tejasphase22 — COMMENTED** [review]
Please check the given comments.

File: src/swapnil/Assignment65.java (hunk 2 above)
Comment (swapnilhole11690) [reply src/swapnil/Assignment65.java:28]: concanate middle part as it is

File: src/swapnil/Assignment63.java (hunk 1 above)
Comment (swapnilhole11690) [reply src/swapnil/Assignment63.java:19]: noted

**Comment by tejasphase22** [comment]
Ok. Got it.

*Tejas Phase*
*+91-9657769508 / 8149220676*

File: src/swapnil/Assignment66.java (hunk 3 above)
Comment (swapnilhole11690) [reply src/swapnil/Assignment66.java:45]: if word have digit goto getDigitWord and get true as return value and print output word without digit 
else 
reverse word

**Pull request merged**